tests/test_all.py

```diff
@@ -273,9 +274,17 @@ def test_no_downgrade_on_install(container: ContainerData) -> None:
     of any package already installed in the container as that would throw
     a question to the user and break the builds.
     """
-
     conn = container.connection
-    conn.run_expect([0], "timeout 2m zypper ref && zypper -n in libsolv-tools")
+
+    zypper_ref = conn.check_output("timeout 2m zypper ref")
+    # bsc1243360: Check for errors in container-suseconnect-zypper. This check must be limited to SLES hosts
+    if "SLES" in LOCALHOST.check_output("cat /etc/os-release"):
```

Comment:
this fails to catch SL Micro or SLE Micro, but I guess this is catching the QA worker right?

Reply:
Currently we anyways have no test runs on SLEM.